drake/systems/plants/collision/FCLModel.h

```diff
@@ -0,0 +1,104 @@
+#ifndef __DrakeCollisionFCLModel_H__
```

Comment:
I copied the style from BulletModel.h I will change FCLModel and leave BulletModel the way it is.
Resolved